internal/resource/cuda-device.go

```diff
@@ -97,6 +97,14 @@ func (d *cudaDevice) IsMigEnabled() (bool, error) {
 	return false, nil
 }
 
+func (d *cudaDevice) IsFabricAttached() (bool, error) {
```

Comment:
Nit: Can we implement this after `GetPCIClass()`?

Reply:
done!